test/unit/math/rev/functor/adj_jac_apply_test.cpp

```diff
@@ -422,17 +422,8 @@ TEST(AgradRev, test_matrix_sin_multiple_jac) {
  * specification.
  */
 struct WeirdArgumentListFunctor1 {
-  template <size_t size>
-  Eigen::VectorXd operator()(
-      std::array<bool, size> needs_adj, double, int, const double&, const int&,
-      std::vector<double>, std::vector<int>, const std::vector<double>&,
-      const std::vector<int>&, Eigen::Matrix<double, Eigen::Dynamic, 1>,
-      Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic>,
-      Eigen::Matrix<double, 2, Eigen::Dynamic>, Eigen::Matrix<double, 5, 1>,
-      const Eigen::Matrix<double, Eigen::Dynamic, 1>&,
-      const Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic>&,
-      const Eigen::Matrix<double, 2, Eigen::Dynamic>&,
-      const Eigen::Matrix<double, 5, 1>&) {
+  template <size_t size, typename... Args>
+  Eigen::VectorXd operator()(std::array<bool, size> needs_adj, Args&&...) {
```

Comment:
I think explicitely specified arguments also serve as a kind of documentation, which types are tested.

Reply:
Yeah that's fair I switched all these back